### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by archydeberker**
Thanks for the awesome package!

This PR fixes some small issues I observed going through the documentation. Some of them might be patches that would be better solved by deleting some docs and redirecting people to information elsewhere.

- There's an inferior version of the README in `website/README`. This was linked from `docs/README`, which I've redirected to the root `README`. Perhaps better to delete entirely

- Fixed inconsistent use of backticks for different algorithms in root `README`

- Added references to specific algorithms for the reference list in root `README`, and deleted any dangling references. It might be better to simply redirect users to `docs/algorithms.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -394,20 +393,18 @@ python captum/insights/example.py
...
-* [Influence-Directed Explanations for Deep Convolutional Networks, Klas Leino et al. 2018](https://arxiv.org/pdf/1802.03788.pdf)
-* [Towards better understanding of gradient-based attribution methods for deep neural networks, Marco Ancona et al. 2018](https://openreview.net/pdf?id=Sy21R9JAW)
+* `NoiseTunnel`: [Sanity Checks for Saliency Maps, Julius Adebayo et al. 2018](https://arxiv.org/abs/1810.03292)
+* `NeuronConductance`: [How Important is a neuron?, Kedar Dhamdhere et al. 2018](https://arxiv.org/abs/1805.12233)
+* `DeepLift`: [Learning Important Features Through Propagating Activation Differences, Avanti Shrikumar et al. 2017](https://arxiv.org/pdf/1704.02685.pdf)
+* `NeuronIntegratedGradients`: [Computationally Efficient Measures of Internal Neuron Importance, Avanti Shrikumar et al. 2018](https://arxiv.org/pdf/1807.09946.pdf)
+* `GradientShap`: [A Unified Approach to Interpreting Model Predictions, Scott M. Lundberg et al. 2017](http://papers.nips.cc/paper/7062-a-unified-approach-to-interpreting-model-predictions)
+* `InternalInfluence`: [Influence-Directed Explanations for Deep Convolutional Networks, Klas Leino et al. 2018](https://arxiv.org/pdf/1802.03788.pdf)
+
```
Comment (NarineK) [inline README.md]: Do you mind also adding:

`Saliency`, `NeuronGradient`: [Deep Inside Convolutional Networks: Visualising
Image Classification Models and Saliency Maps, K. Simonyan, et. al. 2014](https://arxiv.org/pdf/1312.6034.pdf)

**Comment by archydeberker** [comment]
Comments addressed @NarineK!

File: README.md [hunk 2]
```diff
@@ -394,20 +393,18 @@ python captum/insights/example.py
 See the [CONTRIBUTING](CONTRIBUTING.md) file for how to help out.
 
 
-## References
+## References for algorithms mentioned here
 
-* [Axiomatic Attribution for Deep Networks, Mukund Sundararajan et al. 2017](https://arxiv.org/abs/1703.01365)
-* [Did the Model Understand the Question? Pramod K. Mudrakarta, et al. 2018](https://www.aclweb.org/anthology/P18-1176)
```
[the remark this answers is not in the document]
Comment (NarineK) [reply README.md]: Awesome! Thank you! Would you, please, add this one:
[Did the Model Understand the Question? Pramod K. Mudrakarta, et al. 2018](https://www.aclweb.org/anthology/P18-1176)
 to `IntegratedGradients, NeuronIntegratedGradient` group followed by the `[Axiomatic Attribution ... `

**Comment by NarineK** [comment]
LGTM! Thank you, @archydeberker ! Merged the PR!

**Pull request closed without merging**